src/Numerics/Mesh/BrickMesh.jl

```diff
@@ -1086,5 +1086,38 @@ function connectmesh(
     )        # neighbor send ranges into `sendelems`
 end
 
+"""
+    (bndytoelem, bndytoface) = enumerateboundaryfaces!(elemtoelem, elemtobndy, periodicity, boundary)
+
+Update the `elemtoelem` array based on the boundary faces specified in
+`elemtobndy`. Builds the `bndytoelem` and `bndytoface` tuples.
+"""
+function enumerateboundaryfaces!(elemtoelem, elemtobndy, periodicity, boundary)
+    nb = 0
+    for i in 1:length(periodicity)
+        if !periodicity[i]
+            nb = max(nb, boundary[i]...)
+        end
+    end
+    @assert nb <= 6
```

Comment:
```suggestion
    # Limitation of the boundary condition unrolling in the DG kernels
    # (should never be violated unless more general unstructured meshes are used
    # since cube meshes only have 6 faces in 3D, and only 1 bcs is currently allowed
    # per face)
  
    @assert nb <= 6
```